Katello/katello
# Fixes #34496 - Redirect to versionDetails after publish

#### What are the changes introduced in this pull request?

It redirects the user to the CVV details page when the publish completes.



#### What are the testing steps for this pull request?

- Publish a new content view version
- Wait for the modals progress bar to reach 100%
- The user should now be redirected to the contentViewVersionDetails of the freshly created version. 

NOTE: This will not attempt to redirect the user if they exit the modal while the task is unfinished.
